tools/enzyme/gmm.py

```diff
@@ -0,0 +1,27 @@
+import json
+import subprocess
+import tempfile
+from os import listdir
+
+
+def compile():
+    # Nothing to do here. We assume everything is precompiled.
+    return True
```

Comment:
Ideally we'd actually run Enzyme on receiving a `"define"` message. Is this infeasible for similar reasons to Tapenade? Or would it actually be possible to do with Enzyme? I'm really interested in being able to do this, since I've heard that Enzyme is slow and it'd be nice for GradBench to be able to quantify that.

Reply:
No, this should be fine, actually.

Reply:
Done now; although there's room to tune what exactly is measured. I think Enzyme is only a small part of the whole C++ compilation process.

Reply:
Awesome, thanks! Yeah I think after we merge #149 (assuming it looks OK to you) we should add an optional `"timings"` field to the tool's response to `"define"` messages, which would have the same format as for `"evaluate"` (a list of `"name"`/`"nanoseconds"` pairs).